prow/config/istio-private_jobs/istio.yaml

```diff
@@ -0,0 +1,37 @@
+common_jobs_mods: &common_jobs_mods
```

Comment:
if we want to avoid yaml magic, you could add a native `defaults` field that does this. Up to you

Reply:
I like the idea. I am currently doing research into various differents yaml processors, mergers, and templating tools (e.g. kustomize, helm, quicktemplate, ytt, aviator, jsonnet, yq, spruce, etc.) to further simply the configuration.